src/pages/issues.tsx

```diff
@@ -39,18 +39,33 @@ const IssuesPage = (props: { location: Location }): JSX.Element => {
     getSiteData(site).then(setSiteData) // tslint:disable-line:no-expression-statement
   }, [])
 
+  // tslint:disable-next-line: no-expression-statement
+  useEffect(() => {
+    if (siteData === null) {
+      return
+    }
+
+    const opened = sortIssues(siteData.issues.opened)
```

Comment:
Earlier in the file, let's reverse the sort orders by making each of the sortFns negative

```typescript
const sortFns = {
  Recent: (issue: Issue) => -issue.initialReport.timestamp.getTime(),
  Upvotes: (issue: Issue) => -issue.aggregates.upvotes.count,
  Comments: (issue: Issue) => -issue.aggregates.comments.count,
}
```